js/util.js

```diff
@@ -1,3 +1,26 @@
 const isEscapeKey = (evt) => evt.key === 'Escape';
 
-export {isEscapeKey};
+const showAlert = (message) => {
+  const alertContainer = document.createElement('div');
+  alertContainer.style.zIndex = '100';
+  alertContainer.style.position = 'absolute';
+  alertContainer.style.width = '100%';
+  alertContainer.style.left = '50%';
+  alertContainer.style.top = '50%';
+  alertContainer.style.padding = '10px 3px';
+  alertContainer.style.fontSize = '18px';
+  alertContainer.style.lineHeight = '20px';
+  alertContainer.style.textAlign = 'center';
+  alertContainer.style.backgroundColor = 'red';
+  alertContainer.style.transform = 'translate(-50%, -50%)';
```

Comment:
```suggestion
  alertContainer.style = {
      zIndex: '100',
      position: 'absolute',
      ... и тд
  }
```
Можно так, будет короче